src/cart/src/gurt/fault_inject.c

```diff
@@ -43,7 +43,7 @@
 /** max length of argument string in the yaml config file */
 #define FI_CONFIG_ARG_STR_MAX_LEN 4096
 
-#define FI_MAX_FAULT_ID 8192
```

Comment:
it's best now to just remove FI_MAX_FAULT_ID altogether. Previously it was used when each fault was stored in the fixed sized array. Now it does not serve any useful purpose anymore other than declaration here and check in 1 place

Reply:
ok